config/locales/errors/en.yml

```
@@ -14,7 +14,7 @@ en:
       password:
         too_short:
           one: This password is too short (minimum is 1 character)
```

Can we keep the pluralization consistent? Same note would apply for other languages. 

```suggestion
          one: Password must be at least %{count} character long
```

---

Yeah I'd be fine if we use "one" verbatim in the string if that's what you mean.